<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /fs/turing/cvs/copasi_dev/copasi/UI/CQDifferentialEquations.ui,v $ </comment>
<comment>   $Revision: 1.6 $ </comment>
<comment>   $Name: Build-33 $ </comment>
<comment>   $Author: pwilly $ </comment>
<comment>   $Date: 2009/06/19 10:06:06 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQDifferentialEquations</class>
 <widget class="CopasiWidget" name="CQDifferentialEquations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>573</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3ScrollView" name="mpScrollView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>194</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="0" column="4">
      <widget class="QPushButton" name="mpSaveButton">
       <property name="text">
        <string>Save Formula to Disk</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <spacer name="spacer1_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>242</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxFunctions">
       <item>
        <property name="text">
         <string>display name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>expand only kinetic functions</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>expand all functions</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxParameters">
       <item>
        <property name="text">
         <string>display numerical value</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>display name</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabelParameters">
       <property name="text">
        <string>local parameters</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>212</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabelFunctions">
       <property name="text">
        <string>functions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ScrollView</class>
   <extends>QWidget</extends>
   <header>q3scrollview.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">copasiWidget.h</include>
  <include location="global">sstream</include>
  <include location="local">q3scrollview.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>CQDifferentialEquations</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxParameters</sender>
   <signal>activated(int)</signal>
   <receiver>CQDifferentialEquations</receiver>
   <slot>slotUpdateWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFunctions</sender>
   <signal>activated(int)</signal>
   <receiver>CQDifferentialEquations</receiver>
   <slot>slotUpdateWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
